integration/presets/remix.ts

```diff
@@ -7,10 +7,10 @@ const remixNode = applicationConfig()
   .setName('remix-node')
   .useTemplate(templates['remix-node'])
   .setEnvFormatter('public', key => `${key}`)
-  .addScript('setup', 'npm i --prefer-offline')
-  .addScript('dev', 'npm run dev')
-  .addScript('build', 'npm run build')
-  .addScript('serve', 'npm run start')
+  .addScript('setup', 'pnpm i --prefer-offline')
```

Comment:
🙃 Maybe we should remove `--prefer-offline` for consistency?

Reply:
Done!